tools/cordova/project.js

```diff
@@ -403,15 +403,15 @@ from Cordova project`, async () => {
 
     const installedPlatforms = this.listInstalledPlatforms();
 
-    for (platform of platforms) {
+    for (let platform of platforms) {
       if (_.contains(installedPlatforms, platform)) {
         continue;
       }
 
       this.addPlatform(platform);
     }
 
-    for (platform of installedPlatforms) {
+    for (let platform of installedPlatforms) {
```

Comment:
Yep, looks like I missed these in https://github.com/meteor/meteor/pull/8728/commits/26e7cc480a019b522e7c88dba824ed76e5a9e8da. Thanks for fixing!